sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -8,6 +8,7 @@ export enum Role {
   AESTManualTriggerReassessment = "aest-manual-trigger-reassessment",
   AESTReports = "aest-reports",
   AESTCreateInstitution = "aest-create-institution",
+  AESTEditCASSupplierInfo = "aest-edit-cas-supplier-info",
```

Comment:
Is this getting used?

Reply:
This constant should not be part of the PR. It is part of [Andre PR](https://github.com/bcgov/SIMS/pull/3674)
I know without this E2E tests will fail but that will happen only until his code is merged.

Reply:
This role is still not removed.